contracts/libraries/NFTDescriptor.sol

```diff
@@ -113,43 +113,35 @@ library NFTDescriptor {
     }
 
     function formatTokenSymbol(string memory symbol) internal pure returns (string memory) {
```

Comment:
seems this method should be more like `escapeQuotes(string) returns (string)` and could use some fuzz testing (it may not work right multibyte characters, not sure)